ddcz/users.py

```diff
@@ -8,10 +8,10 @@ def migrate_user(profile, password):
     """Create proper Django user for an existing (presumably authenticated) UserProfile"""
     user = User.objects.create_user(
         id=profile.id,
-        username=profile.nick_uzivatele,
+        username=profile.nick,
         password=password,
-        date_joined=profile.reg_schval_datum,
-        last_login=profile.pospristup,
+        date_joined=profile.registration_approved_date,
+        last_login=profile.last_access,
```

Comment:
Access is rather "have the possibility" than real entering the page, isn't it? I like the `last_login` better.

Reply:
Hmm, in this case, I derived access from "accessed", but I can see how this is confusing. Lemme change it since that's how it works now anyway.